daemons/based/based_callbacks.c

```diff
@@ -45,8 +45,6 @@ typedef struct cib_local_notify_s {
 
 int next_client_id = 0;
 
-gboolean legacy_mode = FALSE;
-
```

Comment:
It could be, yeah. But I wonder what's the best way to indicate that we're dropping support for rolling upgrades from < 2.0.0. I used dropping `legacy_mode` as a proxy for that. I'd prefer to do something more specific to version support, before this commit.

Reply:
That can be a documentation commit (the version compatibility table in Pacemaker Administration)

Reply:
yep, no problem documenting 3.0.0 already